clinica/pipelines/statistics_surface/_utils.py

```diff
@@ -0,0 +1,29 @@
+from string import Template
+
+import pandas as pd
+
+MISSING_TERM_ERROR_MSG = Template(
+    "Term ${term} from the design matrix is not in the columns of the "
+    "provided TSV file. Please make sure that there is no typo."
+)
+
+
+def _is_categorical(df: pd.DataFrame, column: str) -> bool:
```

Comment:
I wonder if this module is not a bit overkill for the task it contains. Maybe an inline code like that does the below would suffice ? Or perhaps moving `_is_categorical` to a utils module across pipelines.

```
def is_categorical(array_like):
    return array_like.dtype.name == 'category'
```